styles/globals.css

```diff
@@ -10,6 +10,11 @@ body {
   font-family: 'Urbanist', sans-serif;
 }
 
+::selection{
+  background-color: #714eff;
+  color: lightgray;
```

Comment:
```suggestion
  color: #e5e7eb;
```

Reply:
Hey @GabrielBittar, it looks like there needs to be another change here.

Reply:
Sorry! I'll start using hexcodes only.